.github/workflows/testing.yml

```diff
@@ -68,3 +68,13 @@ jobs:
         check_filenames: true
         check_hidden: true
         skip: ./.git
+
+  run_with_plugin_bugbear:
```

Comment:
Please add this above the `codespell` and name it `Test run action (plugins)` or something. For my sanity :D